src/docs/_includes/navigation.html

```diff
@@ -133,19 +134,19 @@
         }
       }
 
-      if (enter) {
-        var searchResults = $searchResults[0];
-        var href = searchResults.children[active].querySelector('a').getAttribute('href');
+      if (enter && active >= 0) {
+        var href = $searchResults.childNodes[active].querySelector('a').href;
         location.assign(href);
       }
     });
 
-  $searchResults.on('mouseenter', function(e) {
-    var children = $searchResults.children();
+  $searchResults.addEventListener('mouseenter', function(e) {
+    var children = $searchResults.childNodes;
 
     for(var i=0; i<children.length; i++){
```

Comment:
maybe move it to external file and lint? shouldn't affect page loading that much